tests/components/aranet/test_sensor.py

```diff
@@ -92,7 +92,7 @@ async def test_sensors_aranet4(
     co2_sensor = hass.states.get("sensor.aranet4_12345_carbon_dioxide")
     co2_sensor_attrs = co2_sensor.attributes
     assert co2_sensor.state == "650"
-    assert co2_sensor_attrs[ATTR_FRIENDLY_NAME] == "Aranet4 12345 Carbon Dioxide"
+    assert co2_sensor_attrs[ATTR_FRIENDLY_NAME] == "Aranet4 12345 Carbon dioxide"
```

Comment:
I assume the inconsistent casing of "Carbon dioxide" is an artifact of Home Assistant's default translation here? In particular, "Aranet4 12345 carbon dioxide" (sentence case for the whole name) or "[...] Carbon Dioxide" (title case) seem reasonable, but this mixed version feels weird.

If other components end up with the same casing and this ends up being more consistent, I suppose that's fine, I just don't actually know how other components end up looking.

Reply:
This PR is waiting on another one which gives bluetooth support to be able to do transitions. The casing does change a little bit but this is according to HA guidelines.